xml/System.IO/FileSystemInfo.xml

```diff
@@ -189,7 +189,9 @@
   
 -   <xref:System.IO.DirectoryInfo.EnumerateFileSystemInfos%2A>  
   
- To get the latest value, call the <xref:System.IO.FileSystemInfo.Refresh%2A> method.  
+ The value is cached when it or length or time properties are retrieved. To get the latest value, call the <xref:System.IO.FileSystemInfo.Refresh%2A> method.
+
+ If the path does not exist as of the last cached state, the returned value will be `(FileAttributes)(-1)`. <xref:System.IO.FileNotFoundException%2A> or <xref:System.IO.DirectoryNotFoundException%2A> is thrown only when setting the value.
```

Comment:
does not -> doesn't 
returned value -> return value
will be -> is
Remove the %2A from the exception links. links are not working like that. See staging at https://review.docs.microsoft.com/en-us/dotnet-new/api/system.io.filesysteminfo.attributes?view=netframework-4.7&branch=pr-en-us-2209#System_IO_FileSystemInfo_Attributes
Should this information about exception only being thrown when setting the value be moved to the exceptions table? Remarks don't make it into the IntelliSense.